docs/guides/azure-private-link-workspace-simplified.md

```diff
@@ -0,0 +1,310 @@
+---
+page_title: "Provisioning Databricks on Azure with PrivateLink - Simplified deployment"
```

Comment:
```suggestion
page_title: "Provisioning Databricks on Azure with Private Link - Simplified deployment"
```

It is Private Link (two words), let's just make sure it is consistent across this guide